Bottom: 76329407775f254d389b1597594ba1b0fcf6a399
Top:    73bf2bfa87ec558176624bf48bcc8764abed8951
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:35 +0000

Fix undo in MoveObjectAndRubberband

XXX: Need to check if the path which previously missed
     IncrementUndoSerialNumber() was doing so for a reason.


---

diff --git a/src/move.c b/src/move.c
index cfdf881..15cbddc 100644
--- a/src/move.c
+++ b/src/move.c
@@ -946,6 +946,7 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
       MoveLinePoint (Ptr1, moving_line, &moving_line->Point2);
 
       ptr2 = moving_line;
+      IncrementUndoSerialNumber ();
     }
   else
     {
